shared/reducers/fs.js

```diff
@@ -9,6 +9,10 @@ export default function(state: Types.State = initialState, action: FsGen.Actions
   switch (action.type) {
     case FsGen.resetStore:
       return initialState
+    case FsGen.filePreviewLoad:
+      return state
+    case FsGen.filePreviewLoaded:
+      return state.update('metas', metas => metas.set(action.payload.path, action.payload.meta))
```

Comment:
Is `state.metas` necessary? We have the same information in `state.fs.pathItems`; could just update that and use it right?

Reply:
Done (here and other places)